packages/sanity/src/desk/components/confirmDeleteDialog/ConfirmDeleteDialogBody.tsx

```diff
@@ -55,29 +57,31 @@ export function ConfirmDeleteDialogBody({
         <Box padding={2}>
           <SanityDefaultPreview
             icon={UnknownIcon}
-            title="Preview Unavailable"
-            subtitle={`ID: ${item._id}`}
+            title={t('confirm-delete-dialog.preview-item.preview-unavailable.title')}
+            subtitle={t('confirm-delete-dialog.preview-item.preview-unavailable.subtitle', {
+              documentId: item._id,
+            })}
             layout="default"
           />
         </Box>
       )
     },
-    [schema, onReferenceLinkClick],
+    [schema, t, onReferenceLinkClick],
   )
 
   if (internalReferences?.totalCount === 0 && crossDatasetReferences?.totalCount === 0) {
     return (
       <Text as="p">
-        Are you sure you want to {action} <strong>“{documentTitle}”</strong>?
+        <Translate
+          t={t}
+          i18nKey="confirm-delete-dialog.confirmation.text"
+          components={{DocumentTitle: () => <strong>“{documentTitle}”</strong>}}
+          values={{context: action}}
```

Comment:
I would move these quotation marks to the translation. In Norwegian, we use `«` and `»` instead of `“` and `”`, for instance.

```suggestion
          components={{DocumentTitle: () => <strong>{documentTitle}</strong>}}
          values={{context: action}}
```

Then make the key:

```json
{
  "confirm-delete-dialog.confirmation.text_delete":
    "Are you sure you want to delete “<DocumentTitle/>”?"
}
```